### PR TITLE
4.4.0-beta.1-release

### PR DESCRIPTION
**Pull request by xinlian12**
### New Features
* Added new API to efficiently load many documents (via list of pk/id pairs or all documents for a set ok pk values).
* Added new `deleteItem` API.
* Enabled query metrics by default.
### Key Bug Fixes
* Fixed NPE in `GatewayAddressCache`
* Fixing query metric issue for zero item response
* Improved performance (reduced CPU usage) for address parsing and Master-Key authentication

copy of https://github.com/Azure/azure-sdk-for-java/pull/14514 and minor style change

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 4.4.0-beta.1 (2020-08-27)
+### New Features
 * Added new API to efficiently load many documents (via list of pk/id pairs or all documents for a set ok pk values)
 * Enabled query metrics by default
-* Fixed NPE in GatewayAddressCache
+### Key Bug Fixes
+* Fixed NPE in `GatewayAddressCache`
+* Fixing query metric issue for zero item response
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:9]: Improved performance (reduced CPU usage) for address parsing and Master-Key authentication

**Review by FabianMeiswinkel — APPROVED** [review]

**Review by kirankumarkolli — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 # Release History
 
 ## 4.4.0-beta.1 (2020-08-27)
+### New Features
 * Added new API to efficiently load many documents (via list of pk/id pairs or all documents for a set ok pk values)
 * Enabled query metrics by default
-* Fixed NPE in GatewayAddressCache
+### Key Bug Fixes
+* Fixed NPE in `GatewayAddressCache`
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:8]: please put a "." at the end of each changelog entry
to be consistent with other changelog entries:

Comment (moderakh) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:8]:
```suggestion
* Fixed NPE in `GatewayAddressCache`.
```

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md]: Fixed

**Comment by xinlian12** [comment]
> please put a "." at the end of each changelog entry
> to be consistent with other changelog entries.
> 
> other than that LGTM. signed off.

Thanks Mo, added .

**Pull request merged**